client/src/components/Cashflow.js

```diff
@@ -1,11 +1,11 @@
 import { Card, Container } from "react-bootstrap"
 import ExpenseList from "./expenses/ExpenseList";
 import IncomeList from "./income/IncomeList";
-
+// className="mb-3 p-2 shadow-sm" 
```

Comment:
```suggestion

```

kept around because I was experimenting with class params; this isn't needed anymore